Bookish/Views/Book/Catalogue.cshtml

```diff
@@ -19,31 +27,56 @@
             <th>Number of available copies</th>
             <th>Number of searches</th>
         </tr>
-        
+        </thead>
+
+        <tbody>
         @{
+            int i = -1;
+            Book btnBook = new Book();
             if (Model.AllBooks.Count >= 1)
             {
-                foreach (var bookCopy in Model.AllBooks)
+                foreach (var eachBook in Model.AllBooks)
```

Comment:
This loop doesn't need to go within an `if` statement. What would happen if you just had the `foreach` loop, and `Model.AllBooks` had length 0?